aten/src/ATen/native/cuda/UpSampleBilinear2d.cu

```diff
@@ -264,20 +334,32 @@ static void upsample_bilinear2d_backward_out_cuda_template(
       grad_output.scalar_type(), "upsample_bilinear2d_backward_out_frame", [&] {
         using accscalar_t = at::acc_type<scalar_t, true>;
 
-        auto idata = grad_input.packed_accessor<scalar_t, 4>();
-        auto odata = grad_output.packed_accessor<scalar_t, 4>();
+        auto idata = grad_input.data<scalar_t>();
+        auto odata = grad_output.data<scalar_t>();
 
         const accscalar_t rheight = area_pixel_compute_scale<accscalar_t>(
             input_height, output_height, align_corners);
         const accscalar_t rwidth = area_pixel_compute_scale<accscalar_t>(
             input_width, output_width, align_corners);
 
+        const bool fast_atomics = grad_input.is_contiguous();
```

Comment:
Hmm, kernels are definitely written assuming that they operate on contiguous inputs and outputs (no strides are passed as arguments), so is it possible for grad_input to not be contiguous?

Reply:
good catch. Apparently `resize_` from couple lines above does ensure a contiguous tensor storage.
```
In [16]: y.is_contiguous()
Out[16]: False

In [18]: y.resize_(5, 2, 5)

In [21]: y.is_contiguous()
Out[21]: True
```

I'll remove the flag here and add a comment in at the line of `resize_`

Reply:
If you passed a discontiguous grad_input of the correct size here https://github.com/pytorch/pytorch/pull/21879/files#diff-dd2e00b4295272f11ce48eae631e3e49L214, resize would not do anything, but I'm not sure if this situation ever arises, probably not. resize_ on an incorrectly sized tensor would make it contiguous.

Reply:
You are absolutely right. I'll add a `contiguous_()` here.